XenAdmin/Wizards/PatchingWizard/PatchingWizard_SelectServers.cs

```diff
@@ -193,6 +193,15 @@ private void EnabledRow(Host host, UpdateType type, int index)
                     return;
                 }
 
+                var pool = Helpers.GetPool(host.Connection);

```

Comment:
The method Updates.GetUpgradeSequence returns null when the host versions are not the same, in which case we'll go into the first if-block and we won't reach down here to show the right reason. This block here needs to move into the first if-block.